prusti-viper/src/verification_service.rs

```diff
@@ -0,0 +1,12 @@
+use viper;
+use super::encoder::vir::Program;
+
+pub trait VerificationService {
+    fn verify(&self, program: Program, config: ViperBackendConfig) -> viper::VerificationResult;
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum ViperBackendConfig {
+    Silicon(Vec<String>),
+    Carbon(Vec<String>),
+}
```

Comment:
Could you add a brief rustdoc that explains what is a `ViperBackendConfig` and what are the `Vec<String>`?